site/spi/Makefile-bits

```diff
@@ -16,26 +16,18 @@ ifeq ($(SP_OS), rhel7)
 
     ## If not overridden, here is our preferred LLVM installation
     ## (may be changed as new versions are rolled out to the facility)
-    LLVM_DIRECTORY ?= /shots/spi/home/lib/arnold/rhel7/llvm_4.0_final/
+    LLVM_DIRECTORY ?= /shots/spi/home/lib/arnold/rhel7/llvm_4.0
```

Comment:
And similarly, I installed llvm_5.0_RC2, with a llvm_5.0 link that points to it. But in a couple weeks when a final one comes out, I'll just update the link rather than change all the build files.